core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/Project.java

```diff
@@ -42,12 +43,22 @@
  * @author Rob Davey
  * @since 0.0.2
  */
-public interface Project extends com.eaglegenomics.simlims.core.Project, Comparable<Project>, SecurableByProfile,
-    Deletable, Watchable, Nameable {
+public interface Project extends Comparable<Project>, SecurableByProfile, Deletable, Watchable, Nameable, Serializable {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Move constants to a class or enum. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1214)